src/SFML/Network/IpAddress.cpp

```diff
@@ -145,7 +145,7 @@ IpAddress IpAddress::getPublicAddress(Time timeout)
 {
     // The trick here is more complicated, because the only way
     // to get our public IP address is to get it from a distant computer.
-    // Here we get the web page from http://www.sfml-dev.org/ip-provider.php
+    // Here we get the web page from https://www.sfml-dev.org/ip-provider.php
```

Comment:
This can't be changed! 🙀

Reply:
Why not? the site is also reachable with https?

Reply:
`sf::TcpSocket` is unable to do SSL.

Reply:
Yep I mistook it for a documentation in a header.